### PR TITLE
Allow the EA meta box to failover to meta fields when EA record is mia

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/68288

### CODE REVIEW

File: src/admin-views/aggregator/meta-box.php [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-	<dt><?php echo esc_html__( 'Last Import:', 'the-events-calendar' ); ?></dt>
-	<dd>
-		<span class="tribe-value"><?php echo esc_html( $last_import ); ?></span>
-		<span
-			class="dashicons dashicons-editor-help tribe-sticky-tooltip"
-			title="<?php echo esc_attr__( 'The last time this event was imported and/or updated via import.', 'the-events-calendar' ); ?>"
-		></span>
-	</dd>
+	<?php if ( ! empty( $source ) ) : ?>
+		<dt><?php echo esc_html__( 'Source:', 'the-events-calendar' ); ?></dt>
```
Comment (barryhughes) [inline src/admin-views/aggregator/meta-box.php]: Simplify to `esc_html_e()` (ditto line 9)?

File: src/admin-views/aggregator/meta-box.php [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+		<dd><span class="tribe-value"><?php echo esc_html( $source ); ?></span></dd>
+	<?php endif; ?>
+	<?php if ( ! empty( $last_import ) ) : ?>
+		<dt><?php echo esc_html__( 'Last Import:', 'the-events-calendar' ); ?></dt>
+		<dd>
+			<span class="tribe-value"><?php echo esc_html( $last_import ); ?></span>
+			<span
+				class="dashicons dashicons-editor-help tribe-sticky-tooltip"
+				title="<?php echo esc_attr__( 'The last time this event was imported and/or updated via import.', 'the-events-calendar' ); ?>"
```
Comment (barryhughes) [inline src/admin-views/aggregator/meta-box.php]: Simplify to `esc_attr_e()`?

**Comment by borkweb** [comment]
Fixed

**Comment by barryhughes** [comment]
:label:

**Pull request merged**